paper-menu-button.html

```diff
@@ -90,11 +90,14 @@
       margin-top: 20px;
     }
 
-    paper-material {
+    iron-dropdown {
+      @apply(--paper-menu-button-dropdown);
+    }
+
+    .dropdown-content {
       border-radius: 2px;
       background-color: var(--paper-menu-button-dropdown-background, --primary-background-color);
-
-      @apply(--paper-menu-button-dropdown);
+      @apply(--paper-menu-button-content);
```

Comment:
I don't think this is needed -- it's a light dom child, so you can just style it from wherever you're using the `<paper-menu-button>`